.github/workflows/ci.yml

```diff
@@ -34,6 +34,13 @@ jobs:
           registry: private.docker.xenit.eu
           username: ${{ secrets.CLOUDSMITH_USER }}
           password: ${{ secrets.CLOUDSMITH_APIKEY }}
+      # REquired as long as jodconverter is required and not moved to SW
```

Comment:
What is SW?

Reply:
I'm thinking maybe Scaleway? I think you mean Cloudsmith.
Can you change that before committing? I'll already approve.